frontend/packages/dev-console/integration-tests/features/topology/topology-smoke_test

```diff
@@ -0,0 +1,150 @@
+Feature: Topology smoke test
+	As a user, I want see a page for Pipeline
+
+Background:
+    Given user logged into the openshift application
+    
+
+@regression, @smoke
+Scenario: Empty state of topology
+   Given user is in administrative perspective
+   When user goes to developer perspective
+   And user selects a existing project from project list or create a new project
```

Comment:
Try to avoid multiple conditions in one statement
eg: And user selects a existing project from project list
As our scenario is to check the topology empty state